README.md

```diff
@@ -68,6 +68,12 @@ In addition you would need
     node authentication_sample.js
     ```
 
+## Reference samples
+
+- [helloWorld.js](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-secrets/samples/javascript/helloWorld.js)
+- [ARM Key Vault README Sample](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/keyvault/arm-keyvault#sample-code)
+- [ARM Resources README Sample](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/resources/arm-resources#sample-code)
```

Comment:
I would recommend removing the above section and re-using the next section which is `References and further reading`

- Instead of hello world sample, I would recommend pointing to the sample folder so that reader can see other samples
- The readme sample pointers for arm-keyvault and arm-resources dont provide more information than what is already shared in this repo, so I would skip them altogehter